docs/charts/index.yaml

```diff
@@ -2,65 +2,65 @@ apiVersion: v1
 entries:
   spring-config-server:
   - apiVersion: v1
-    created: 2017-09-26T15:09:20.739020728-05:00
+    created: 2017-10-03T10:02:49.440768-04:00
     description: Spring Config Server
-    digest: c43bb0fa34431eea19bc8528f6b864ebc22ebc5f1c426f2cad3104e07125d025
+    digest: 0b011aaba6d7c242957ecd81d0723641ed55077d979c7acc8c3f37c4608a70d2
     name: spring-config-server
     urls:
-    - https://raw.githubusercontent.com/ibm-cloud-architecture/refarch-cloudnative-spring/master/docs/charts/spring-config-server-1.0.0.tgz
+    - http://192.168.0.134:8879/spring-config-server-1.0.0.tgz
```

Comment:
@jkwong888 was this intentional? If not, can you make sure they are packaged with the right url?